src/containers/product-details/index.js

```diff
@@ -87,63 +88,71 @@ const ProductDetailsArea = memo(
       setShowBuyLightningModal((prev) => !prev);
     };
 
+    const getProperties = () => {
+      if (!inscription) {
+        return [];
+      }
+      return [
+        {
+          id: 8,
+          type: "Owner",
+          value: shortenStr(inscription.owner),
+        },
+        {
+          id: 7,
+          type: "Inscription Id",
+          value: shortenStr(inscription.id),
+        },
+
+        {
+          id: 1,
+          type: "Content Type",
+          value: inscription.content_type,
+        },
+        {
+          id: 2,
+          type: "Content Length",
+          value: `${inscription.content_length} bytes`,
+        },
+
+        {
+          id: 4,
+          type: "Genesis Height",
+          value: inscription.genesis_height,
+        },
+        {
+          id: 5,
+          type: "Genesis Fee",
+          value: inscription.genesis_fee,
+        },
+        {
+          id: 6,
+          type: "Number",
+          value: inscription.num,
+        },
+        {
+          id: 3,
+          type: "Created",
+          value: minted,
+        },
+      ];
+    };
+
     const [isOwner, setIsOwner] = useState(false);
 
     useEffect(() => {
       setIsOwner(
         nostrOrdinalsAddress &&
           inscription.owner &&
-          nostrOrdinalsAddress === inscription.owner
+          nostrOrdinalsAddress === inscription.owner,
```

Comment:
Maybe something wrong here?

Reply:
no theproblem is that inscription is undefifined when the effect gets executed, since we are doing the skeleton this is expected, I just need to validate that there is an inscription in the first place